schema/sql/api.sql

```diff
@@ -31,9 +31,23 @@ grant usage on sequence "user_id" to "api-user-create";
 grant "api-user-create" to "api";
 grant "api-anon" to "api-user-create";
 
+create role "api-user-manage";
+comment on role "api-user-manage" is
+    $$Intended for use with user management systems$$;
+grant usage on sequence "user_id" to "api-user-manage";
+grant select,insert,update on table
+    public."passwd",
+    public."ssh_public_key",
+    public."openpgp_public_key"
+to "api-user-manage";
```

Comment:
These lines don't belong here: the permissions should be with the tables; not with the user.

Reply:
Then why do nearly identical lines live with "api".

I am just continuing the existing pattern

Reply:
`api` is special: it's the master role used to pivot to the others.

Reply:
there are no references to api-user-create anywhere but api.sql.

Having to hunt across every SQL file to find out what permissions a given role has seems a bit silly. We don't do this for anything else.

Reply:
You should be able to remove all of these lines. Your user should never use the public schema directly (aside from the sequence which you inherit via api-anon).
You access these 3 tables via the views below.

Reply:
Yeah you are right. tests still pass. Removed.